### PR TITLE
Split locations into primary and replica

### PR DESCRIPTION
**Pull request by jtweed**

### CODE REVIEW

File: docs/rfcs/002-archival_storage/README.md [hunk 1]
```diff
@@ -555,28 +555,28 @@ Response:
...
-      "type": "Location",
-      "provider": {
-        "type": "Provider",
-        "id": "aws-s3-ia",
-        "label": "AWS S3 - Infrequent Access"
-      },
-      "bucket": "bucketname-access",
-      "path": "digitised/b24923333",
-      "url": "http://bucketname-access.s3-eu-west-1.amazonaws.com/digitised/b24923333"
+  "primaryLocation": {
```
Comment (jtweed) [inline docs/rfcs/002-archival_storage/README.md]: Any opinions on whether this should be `location` or `primaryLocation`?

Comment (alexwlchan) [reply docs/rfcs/002-archival_storage/README.md]: I would say `location`. The name of the bucket is `wc-storage`, not `wc-storage-primary`.

Comment (kenoir) [reply docs/rfcs/002-archival_storage/README.md]: I think I also prefer `location` for the same reason.

Comment (jtweed) [reply docs/rfcs/002-archival_storage/README.md]: Ok will change 👍

**Pull request merged**